cloud/linode/instances.go

```diff
@@ -108,15 +147,24 @@ func (i *instances) CurrentNodeName(_ context.Context, hostname string) (types.N
 }
 
 func (i *instances) InstanceExistsByProviderID(ctx context.Context, providerID string) (bool, error) {
+	ctx = sentry.SetHubOnContext(ctx)
+	sentry.SetTag(ctx, "provider_id", providerID)
+
 	id, err := linodeIDFromProviderID(providerID)
 	if err != nil {
+		sentry.CaptureError(ctx, err)
 		return false, err
 	}
+
+	sentry.SetTag(ctx, "linode_id", id)
+
 	_, err = linodeByID(ctx, i.client, id)
 	if err == nil {
 		return true, nil
 	}
 
+	// TODO: Should we capture an error here?
```

Comment:
since we're not capturing errors for any of the un-exported methods here, yes, let's capture this error

Reply:
Done!